requirements.txt

```diff
@@ -8,7 +8,7 @@ julius
 num2words
 numpy
 sentencepiece
-spacy==3.5.2
+spacy==3.6.1
```

Comment:
I want this to, but to keep backwards compat, is it possible to keep it at spacy>=3.5.2?

Reply:
Sure, changed to the soft limit. Also looked at the Spacy upgrading guide [here](https://spacy.io/usage/v3-6#upgrading) and it doesn't look like there should be a problem. Current code doesn't use Spacy cfg files or `Span` objects.